test/math/FloatVectorListTest.ooc

```diff
@@ -144,6 +144,23 @@ FloatVectorListTest: class extends Fixture {
 			list1 free()
 			list2 free()
 		})
+		this add("scalar product", func {
+			list1 := FloatVectorList new()
+			list2 := FloatVectorList new()
+			list1 add(1.0f)
+			list1 add(2.0f)
```

Comment:
The reason it's not like that elsewhere in the file is because this is a very new concept and I've prioritized fixing files in source over tests. The tests will be fixed eventually though, so I'd rather see people getting used to this approach.